onedal/cluster/tests/test_kmeans_init.py

```diff
@@ -0,0 +1,91 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import pytest
+import numpy as np
+
+from numpy.testing import assert_array_equal
+from daal4py.sklearn._utils import daal_check_version
+
+if daal_check_version((2023, 'P', 200)):
+    from onedal.cluster import kmeans_plusplus, KMeans
+    from onedal.tests.utils._device_selection import get_queues
+
+    from sklearn.datasets import load_breast_cancer
+    from sklearn.metrics import davies_bouldin_score
+
+    @pytest.mark.parametrize('queue', get_queues())
+    @pytest.mark.parametrize('dtype', [np.float32, np.float64])
+    @pytest.mark.parametrize('n_cluster', [2, 5, 7, 11, 15, 128])

```

Comment:
Too many cases
I think extended testing might be enabled through specific flag which is not enabled by default

Reply:
Fixed